tools/make-nightly.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -eux
+
+cargo build --release --all-features
+
+rm -f out/crucible-nightly.tar.gz 2> /dev/null
+
+mkdir -p out
+
```

Comment:
will this `rm` command error (halting the script) if `out/crucible-nightly.tar.gz` doesn't exist already, or does `-f` prevent that?

Reply:
Yeah, the `-f` command will tell `rm` to not report error.

```
alan@atrium$ rm this_does_not_exist
rm: this_does_not_exist: No such file or directory
alan@atrium:crucible$ echo $?
2
alan@atrium$ rm -f this_does_not_exist
alan@atrium$ echo $?
0
```

Reply:
neat!